### DISCUSSION TITLE
Bulk Selection in Event Sheet

### DISCUSSION THREAD

**Discussion by MattCPinto** [4ian/GDevelop#6985] [Enhancement ideas (and their technical discussions)]
## Description

I would like a double-click bulk selection feature for actions and conditions in the event sheet. When working in the GDevelop interface, users can double-click and hold their mouse to select multiple items. This approach would improve efficiency and simplify managing events and conditions.

User-Friendly: Intuitive double-click interaction aligns with common behavior when working with events, when you double click and hold nothing happens, giving a perfect opportunity to put this feature in.
I hope you guys would like this feature, I sure do, it would help my efficiency working in the event sheet a lot.

## Solution suggested

When working in the GDevelop interface, users can double-click and hold their mouse to select multiple items. This approach would improve efficiency and simplify managing events and conditions. Double click and hold nothing happens, giving a perfect opportunity to put this feature in.

## Alternatives considered

Holding down the mouse button on a blank area of the Event sheet to select multiple events.

**Comment by jayare5** [comment]
And what about the left click an action, shift + click another action at the bottom so that it selects everything in between?
I'm surprised it doesn't work this way. It's very tedious.
